mobile/salto/src/stores.ts

```diff
@@ -1,8 +1,11 @@
-import { readable, type Readable } from 'svelte/store';
+import { readable, writable, type Readable } from 'svelte/store';
 import { platform } from "@tauri-apps/plugin-os";
 
 const currentPlatform = platform();
 
 export const isMobile = readable(false, (set) => {
   set(currentPlatform == "ios" || currentPlatform == "android");
 });
+
+export const selectedCurrencyStored = writable<string>('usd');
```

Comment:
`writable<string | null>(null)` for the case we cannot reach the server.
Also make sure that the price in `selectedCurrencyStored` is only displayed if `selectedCurrencyStored` is not null. If it is null (server not running) don't display it.